src/modules/createTasks.js

```diff
@@ -0,0 +1,107 @@
+const toDoList = document.querySelector('#todo-lists');
+const clearBtn = document.querySelector('button');
+
+export default class CreateTasks {
+    static createTask = () => {
+      const task = document.createElement('div');
+      task.className = 'todo-list-container';
+      toDoList.appendChild(task);
+      const checkbox = document.createElement('input');
+      checkbox.type = 'checkbox';
+      const content = document.createElement('p');
+      content.className = 'content';
+      const dots = document.createElement('i');
+      dots.className = 'dots fa-solid fa-ellipsis-vertical';
+      const trash = document.createElement('i');
+      trash.className = 'alt-fa fa-solid fa-trash';
+
+      task.append(checkbox, content, dots, trash);
+
+      checkbox.addEventListener('click', () => {
+        task.classList.toggle('checked-holder');
+        content.classList.toggle('line-through');
+        trash.classList.toggle('trash-active');
+        dots.classList.toggle('dots-remove');
+
+        const localData = JSON.parse(localStorage.getItem('todo'));
+        const filter = [];
+        const taskChecked = document.querySelectorAll('.todo-list-container');
+        for (let i = 0; i < localData.length; i += 1) {
+          if (taskChecked[i].classList.contains('checked-holder')) {
+            localData[i].completed = true;
+          } else {
+            localData[i].completed = false;
+          }
+
+          filter.push(localData[i]);
+          localStorage.setItem('todo', JSON.stringify(filter));
+        }
+      });
+
+      clearBtn.addEventListener('click', () => {
+        const localData = JSON.parse(localStorage.getItem('todo'));
+        const checkedTasks = document.querySelectorAll('.checked-holder');
+        const filter = [];
+        for (let i = 0; i < checkedTasks.length; i += 1) {
+          toDoList.removeChild(checkedTasks[i]);
+        }
+        for (let i = 0; i < localData.length; i += 1) {
+          if (localData[i].completed !== true) {
+            filter.push(localData[i]);
+          }
+        }
+        localStorage.setItem('todo', JSON.stringify(filter));
+      });
+
+      trash.addEventListener('click', () => {
+        toDoList.removeChild(task);
+        const localData = JSON.parse(localStorage.getItem('todo'));
+        const filteredResult = localData.filter((item) => item.description === content.textContent);
+        const filter = [];
+
+        for (let index = 0; index < localData.length; index += 1) {
+          if (filteredResult[0].description !== localData[index].description) {
+            filter.push(localData[index]);
+          }
+        }
+
+        localStorage.setItem('todo', JSON.stringify(filter));
+      });
+
```

Comment:
[OPTIONAL]
- [ ] Upon deletion, the indexes are not updated. Kindly have your indexes updated upon user actions.

![image](https://user-images.githubusercontent.com/29071984/193139706-c07af3ce-1021-4911-a69a-df6d15885922.png)